ncm-sysctl/src/test/perl/configure.t

```diff
@@ -7,9 +7,8 @@ use Test::More;
 use Test::Quattor qw(simple);
 use NCM::Component::sysctl;
 use Test::MockModule;
-use CAF::Object;
 
-$CAF::Object::NoAction = 1;
+#$CAF::Object::NoAction = 1;
```

Comment:
Why leave it in commented out? Just delete it if it's not needed?

Reply:
Oh whoops, I thought I had.